src/Controls/src/Core/PlatformConfiguration/ExtensionPoints.cs

```diff
@@ -1,16 +1,42 @@
-
 namespace Microsoft.Maui.Controls.PlatformConfiguration
 {
-	/// <include file="../../../docs/Microsoft.Maui.Controls.PlatformConfiguration/Android.xml" path="Type[@FullName='Microsoft.Maui.Controls.PlatformConfiguration.Android']/Docs/*" />
+	/// <summary>
+	/// Marker class that identifies the Android platform.
+	/// </summary>
+	/// <remarks>
+	/// Developers specifiy the type name of this marker class to the <see cref="M:Microsoft.Maui.Controls.IElementConfiguration`1.On``1" /> method to specify the underlying Android control on which to run a platform-specific effect.
```

Comment:
```suggestion
	/// Developers specify the type name of this marker class to the <see cref="M:Microsoft.Maui.Controls.IElementConfiguration`1.On``1" /> method to specify the underlying Android control on which to run a platform-specific effect.
```
Nit: technically we can replace `M:Microsoft.Maui.Controls.IElementConfiguration`1.On``1` with just the actual type simply be removing `M:` in this case. And maybe some of the namespace because its already in the usings. That way it will also break at compile-time if types change or something.